### PR TITLE
Add parsers GrubbyDefaultKernel

### PR DESCRIPTION
**Pull request by xiangce**

### CODE REVIEW

File: insights/parsers/grubby.py [hunk 1]
```diff
@@ -2,29 +2,166 @@
...
+        index=0
+        kernel=/vmlinuz-2.6.32-754.9.1.el6.x86_64
+        args="ro rd_NO_LUKS rd_NO_LVM LANG=en_US.UTF-8 rd_NO_MD SYSFONT=latarcyrheb-sun16 crashkernel=auto  KEYBOARDTYPE=pc KEYTABLE=us rd_NO_DM rhgb quiet"
+        root=UUID=1b46779e-4fae-442d-a2ac-dd6d8563ff3e
+        initrd=/boot/initramfs-2.6.32-754.9.1.el6.x86_64.img
+        index=1
+        kernel=/vmlinuz-2.6.32-573.el6.x86_64
+        args="ro rd_NO_LUKS rd_NO_LVM LANG=en_US.UTF-8 rd_NO_MD SYSFONT=latarcyrheb-sun16 crashkernel=auto  KEYBOARDTYPE=pc KEYTABLE=us rd_NO_DM rhgb quiet"
+        root=UUID=1b46779e-4fae-442d-a2ac-dd6d8563ff3e
+        initrd=/boot/initramfs-2.6.32-573.el6.x86_64.img
```
Comment (lonicerae) [inline insights/parsers/grubby.py]: Please note that the kernel can be reused multiple times in different entry, for example:

```
boot=/dev/sda
index=0
kernel=/vmlinuz-2.6.32-431.el6.x86_64
args="ro rd_NO_LUKS LANG=en_US.UTF-8 rd_NO_MD rd_LVM_LV=VolGroup/lv_swap SYSFONT=latarcyrheb-sun16 crashkernel=auto rd_LVM_LV=VolGroup/lv_root  KEYBOARDTYPE=pc KEYTABLE=us rd_NO_DM"
root=/dev/mapper/VolGroup-lv_root
initrd=/boot/initramfs-2.6.32-431.el6.x86_64.img
index=1
kernel=/vmlinuz-2.6.32-431.el6.x86_64
args="ro rd_NO_LUKS LANG=en_US.UTF-8 rd_NO_MD rd_LVM_LV=VolGroup/lv_swap SYSFONT=latarcyrheb-sun16 crashkernel=auto rd_LVM_LV=VolGroup/lv_root  KEYBOARDTYPE=pc KEYTABLE=us rd_NO_DM quiet"
root=/dev/mapper/VolGroup-lv_root
initrd=/boot/initramfs-2.6.32-431.el6.x86_64.img
```
In this case we may need to rethink the `__getitem__()` method.

Comment (xiangce) [reply insights/parsers/grubby.py]: The `__getitem__` is just used to get specified kernel entry, but not the booting entry.
To get the next booting entry, need to corporate with the GrubbyDefaultKernel which indicates the next booting kernel.

Comment (xiangce) [reply insights/parsers/grubby.py]: Oh, got your point

Comment (xiangce) [reply insights/parsers/grubby.py]: @lonicerae - Thanks for reviewing. I removed the GrubInfoALL parser for now, please check it again.

**Pull request merged**